SermonDetailViewController.m

```diff
@@ -50,85 +126,113 @@ - (void)didReceiveMemoryWarning {
 
 - (IBAction)playSermon:(id)sender {
 
-    MPMoviePlayerViewController *controller = [[MPMoviePlayerViewController alloc]initWithContentURL: [NSURL URLWithString: @"https://bit.ly/2mD7KMJ"]];
-    [controller.moviePlayer prepareToPlay];
-    [controller.moviePlayer play];
-    
-    [self.navigationController presentMoviePlayerViewControllerAnimated:controller];
-    
-    /*
-    if (playerStarted) {
-
-        if (playing) {
-            
-            [self pause];
-            
-        } else {
-            
-            [self play];
-            
-        }
-    } else {
-        
-        [self showLoading];
-        [self.playButton setHidden:YES];
-        [self initiatePlayer];
+    if ([audioPlayer isPlaying]) {
+        [self pause];
+        [NSTimer scheduledTimerWithTimeInterval:1.0 target:self selector:@selector(countUp:) userInfo:nil repeats:YES];
         
+    } else {
+        [self play];
+        [NSTimer scheduledTimerWithTimeInterval:1.0 target:self selector:@selector(countUp:) userInfo:nil repeats:YES];
     }
-     */
+    
+}
+
+-(void)countUp:(NSTimer *) aTimer {
```

Comment:
It looks like `aTimer` is not being used at all in this function. Can we remove it?